uvicorn/protocols/utils.py

```diff
@@ -8,11 +8,11 @@ def get_remote_addr(transport):
         family = socket_info.family
         if family in (socket.AF_INET, socket.AF_INET6):
             return (str(info[0]), int(info[1]))
-        return None
+        return "-"
     info = transport.get_extra_info("peername")
     if info is not None and isinstance(info, (list, tuple)) and len(info) == 2:
         return (str(info[0]), int(info[1]))
-    return None
```

Comment:
Is there a reason why we're changing the `None` case here?

Reply:
`None` was displayed in access log.

```
2019-10-08 15:47:52,870 - INFO - None - "GET / HTTP/1.1" 200
2019-10-08 15:48:00,546 - INFO - None - "GET / HTTP/1.1" 200
2019-10-08 15:48:01,285 - INFO - None - "GET / HTTP/1.1" 200
```

Or we can fix here

```diff
diff --git a/uvicorn/protocols/http/httptools_impl.py b/uvicorn/protocols/http/httptools_impl.py
index 2e1dd90..8b414b2 100644
--- a/uvicorn/protocols/http/httptools_impl.py
+++ b/uvicorn/protocols/http/httptools_impl.py
@@ -436,7 +436,7 @@ class RequestResponseCycle:
             if self.access_log:
                 self.logger.info(
                     '%s - "%s %s HTTP/%s" %d',
-                    self.scope["client"],
+                    self.scope["client"] or "-",
                     self.scope["method"],
                     get_path_with_query_string(self.scope),
                     self.scope["http_version"],
```

Any good idea?